### PR TITLE
Removes tablets (not PDAs) entirely.

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

**Comes with an UpdatePaths!**

Removes the tablet subtype, PDAs now replaces them entirely.

Nukie and Silicon tablets are now subtypes of the PDA instead, while contractor ones were removed entirely as they didn't do anything and were unused (though it wouldn't be hard to re-add).

Nukie PDAs are now the only type of PDA that uses modular_tablets.dmi, which is just larger icons of modular_pda. Each application requires an icon state in both of these, for 2 different sizes, which makes it annoying to make new applications, especially if it can also run on computers/laptops.

### Icons

Because Silicon tablets are now a subtype of PDA, they use PDA icons instead of tablet ones. Luckily for us, they already exist in code.
image

AI's don't use a tablet icon though, so they aren't affected.

## Why It's Good For The Game

There's very little difference between tablets and PDAs, PDAs overshadow them in every single way, so at this point I don't see why we should have both of these, and if you compare the two in usefulness and actual in-game use by players, it's a no-brainer than the item all players get roundstart and comes with a messenger should be the one we go with.

Also as said in the about section, when making an app you would need to make icon states for the program running for all hardware it can run on, which is Computer, Laptop, PDA, and Tablet.

Laptop is just a smaller computer icon
PDA is just a smaller tablet icon

However, you can't simply shrink the size of the icon, instead you have to completely resprite the same app icon FOUR TIMES for it to not bluescreen on all these different devices.

<details>
<summary>
Here's examples of it
</summary>
Computer (NOTE: *They share the same icon file as regular computers*)

Laptop

Tablet

PDA

</details>

If we wish to help in simplifying this, we should remove tablet icons entirely, which means 1 less icon to worry about. To do this, we'd need to resprite nukie PDAs, however I am very much not a spriter and never tried GAGS, so I'll leave it to someone else to do.

## Changelog

:cl:
del: Tablets are now removed, PDAs are now the base 'tablet'. Silicon and nukie tablets are now PDAs.
/:cl:

### CODE REVIEW

File: tools/UpdatePaths/Scripts/71507_tablet_to_pda.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# updates pda subtypes to their new type, and deletes base pdas as they were removed.
+
+/obj/item/modular_computer/tablet/pda : @DELETE
```
Comment (Mothblocks) [inline tools/UpdatePaths/Scripts/71507_tablet_to_pda.txt:3]: Not /obj/item/modular_computer/pda?

Comment (JohnFulpWillard) [reply tools/UpdatePaths/Scripts/71507_tablet_to_pda.txt:3]: oh right i wanted to delete base tablets, my bad

**Comment by distributivgesetz** [comment]
okay i should leave tablets alone for now in my refactor lmao

**Comment by zxaber** [comment]
Sad, but the removal of modular parts pretty much smashed any reason to have the two be different.

We should just replace the nukie tablet with a PDA (or, really, just give it a PDA sprite because that's about the extent of the change).

**Comment by Bawhoppen** [comment]
RIP PDAs I'll always remember you

**Comment by Kapu1178** [comment]
there's something hilarious to be said about an entire modular computer system being merged only for it to be basically rolled back over the coming months.

**Comment by JohnFulpWillard** [comment]
How was it rolled back? We just integrated one into the other

**Comment by Kapu1178** [comment]
> How was it rolled back? We just integrated one into the other

i was referring to removing the "modular" part of modular computers.

No, the programs aren't the modular part. Lmfao.

**Comment by Mothblocks** [comment]
yes they are

**Pull request merged**